src/Result/FormatterInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+namespace ShipMonk\ComposerDependencyAnalyser\Result;
+
+use ShipMonk\ComposerDependencyAnalyser\CliOptions;
+use ShipMonk\ComposerDependencyAnalyser\Config\Configuration;
+
+interface FormatterInterface
```

Comment:
We try to avoid Interface suffixes. This should be `ResultFormatter` and the old implementor probably `ConsoleFormatter`